libmatroska2/minilzo/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 # minilzo library API
 
 add_library("minilzo" STATIC)
+if (BUILD_SHARED_LIBS)
+    set_property(TARGET "minilzo" PROPERTY POSITION_INDEPENDENT_CODE ON)
+endif()

```

Comment:
This contradicts the previous line where "minilzo" is only compiled as a `STATIC` library.

Reply:
Does it? The idea is to build `minilzo` as a static library, but still as PIC, because that static library gets incorporated into the `matroska2` shared library.